doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -234,7 +234,7 @@ reference "input artefacts"
 
 \end_inset
 
-) are integer.
+) are integers.
```

Comment:
[“of integrity” seems to be the correct translation](http://dict.leo.org/ende/index_de.html#/search=integer&searchLoc=0&resultOrder=basic&multiwordShowSingle=on)

Sorry for the confusion! Didn’t laugh so hard for a long time.